tests/regression/mmdet.yml

```diff
@@ -169,7 +169,7 @@ models:
   - name: YOLOV3
     metafile: configs/yolo/metafile.yml
     model_configs:
-      - configs/yolo/yolov3_d53_320_273e_coco.py
+      - configs/yolo/yolov3_d53_8xb8-320-273e_coco.py
```

Comment:
hasn't it changed in https://github.com/open-mmlab/mmdeploy/pull/1334?

Reply:
don't know why GitHub detect the change